openupgrade_scripts/scripts/event_sale/17.0.1.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,21 @@
+---Models in module 'event_sale'---
+---Fields in module 'event_sale'---
+event_sale   / event.event.ticket       / _order                        : _order is now 'event_id, sequence, price, name, id' ('event_id, price')
+# NOTHING TO DO
+
+event_sale   / event.event.ticket       / price_incl (float)            : NEW hasdefault: compute
+# NOTHING TO DO: not sure if needed to precreate it.
```

Comment:
If you have a lot of events, this will be heavy, but let's keep it for now until someone feels is too slow.